### PR TITLE
Added optional language parameter for AnalyzeImageAsync

### PR DESCRIPTION
**Pull request by mrrenaud**
Allows to specify the language in which the description should be localized. For the moment, only English ("en") seems to be supported.

This documentation says "en" and "zh" are supported, though it looks like only "en" is supported for the moment: https://westus.dev.cognitive.microsoft.com/docs/services/56f91f2d778daf23d8ec6739/operations/56f91f2e778daf14a499e1fa

This FAQ says there are even more supported languages for Computer Vision, but it's incorrect: https://docs.microsoft.com/fr-be/azure/cognitive-services/Computer-vision/FAQ

### CODE REVIEW

File: ClientLibrary/VisionServiceClient.cs [hunk 1]
```diff
@@ -485,6 +486,16 @@ private string DetailsToString(IEnumerable<string> details)
...
         }
 
+        /// <summary>
+        /// Converts a shortcode to a querystring
+        /// </summary>
+        /// <param name="language">String containing the language shortcode</param>
+        /// <returns>The language query parameter string</returns>
+        private string LanguageToString(string language)
+        {
+            return language == null ? "" : "language=" + language;
```
Comment (cthrash) [inline ClientLibrary/VisionServiceClient.cs]: Consider string.IsNullOrEmpty(s) instead of null check

Comment (MrRenaud) [inline ClientLibrary/VisionServiceClient.cs]: Thanks, I just pushed the modification.

**Comment by chsienki** [comment]
Yes, should be fine. It's a non-breaking compilation change so no problems with upgrade. It won't be binary drop in replacement (because the method signature has changed), but that's not a guarantee we've ever made.

**Comment by cthrash** [comment]
This should be working as of [this](https://github.com/Microsoft/Cognitive-Vision-Windows/pull/40).  Apologies for not jumping on this sooner.

**Pull request closed without merging**